udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step2DescribeOrganization.vue

```diff
@@ -0,0 +1,364 @@
+              @blur="vWarning$.name.$touch"
+            >
+              <InputGroup
+                data-testid="nameInput"
+                :aria-describedby="nameOrganizationAccordionId"
+                :label="t('Name')"
+                :required="true"
+                v-model="organization.name"
+                :hasError="fieldHasError('name')"
+                :hasWarning="fieldHasWarning('name')"
+                :errorText="getErrorText('name')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addSiretAccordionId"
+            >
+              <InputGroup
+                data-testid="siretInput"
+                :label="t('SIRET Number')"
+                v-model="organization.business_number_id"
+                :hasError="fieldHasError('business_number_id')"
+                :hasWarning="fieldHasWarning('business_number_id')"
+                :errorText="t('This SIRET is not valid')"
+              />
+            </LinkedToAccordion>
+            <div v-if="checkOrga.exists !== null" class="fr-col fr-mx-2v fr-mb-2v bg-contrast-grey text-align-center">
+              <div v-if="checkOrga.exists">
+                <p>{{ t('The SIREN n° {number} is matching', { number: checkOrga.siren }) }}</p>
+                <p>{{ checkOrga.name }}</p>
+                <p v-if="checkOrga.isPublicService">
+                  <span class="fr-icon-bank-line" aria-hidden="true"></span>
+                  {{ t('Public Service') }}
+                </p>
+              </div>
+              <div v-else>
+                <p>{{ t('No organization found matching this SIRET on annuaire-entreprises.data.gouv.fr') }}</p>
+              </div>
+            </div>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addAcronymAccordionId"
+            >
+            <InputGroup
+                data-testid="acronymInput"
+                :label="t('Acronym')"
+                v-model="organization.acronym"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addDescriptionAccordionId"
+              @blur="vWarning$.description.$touch"
+            >
+            <InputGroup
+                data-testid="descriptionInput"
+                :label="t('Description')"
+                :required="true"
+                type="textarea"
+                v-model="organization.description"
+                :hasError="fieldHasError('description')"
+                :hasWarning="fieldHasWarning('description')"
+                :errorText="getErrorText('description')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addWebsiteAccordionId"
+            >
+            <InputGroup
```

Comment:
Also, not related to your code but there is an error with the input `aria-describedby`. It shows ` [object Object]` instead of ids.